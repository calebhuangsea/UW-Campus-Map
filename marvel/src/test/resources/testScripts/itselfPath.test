# Find a path with a node to itself using itselfPath.csv

LoadGraph itselfPath itselfPath.csv

FindPath itselfPath n1 n1