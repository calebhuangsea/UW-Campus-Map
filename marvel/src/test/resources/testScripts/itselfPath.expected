# Find a path with a node to itself using itselfPath.csv

loaded graph itselfPath

path from n1 to n1: